lms/djangoapps/email_marketing/signals.py

```diff
@@ -218,3 +222,14 @@ def _create_sailthru_user_vars(user, profile):
         sailthru_vars['country'] = unicode(profile.country.code)
 
     return sailthru_vars
+
+
+def _get_current_site():
+    """
+    Returns the site for the current request if any.
+    """
+    request = crum.get_current_request()
+    if not request:
+        return
```

Comment:
Looks like we need to cover this line. https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/29207/Diff_Coverage_Report/